docs/release-notes/highlights-8.0.0.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[release-highlights-8.0.0]]
+== 8.0.0 release highlights
+++++
+<titleabbrev>8.0.0</titleabbrev>
+++++
+
+coming[8.0.0]
+
+See also <<breaking-changes-8.0>> and <<release-notes>>.
```

Comment:
Suggest  <<breaking-changes-8.0, breaking changes>> otherwise this renders as "8.0"

Reply:
Good catch, thanks!